malcolm/modules/ADPandABlocks/doublebuffer.py

```diff
@@ -238,7 +239,19 @@ def configure(self, rows_generator: Iterator[SequencerRows]) -> None:
             # The template sets repeats to 1 however older designs may not have
             # been updated.
             for table in self._table_map.values():
-                table.repeats.put_value(1)
+
+                if table.repeats.value != 1:
+                    logging.warning(
+                        "Repeats on Seq table {} should be set to '1', Malcolm has"
```

Comment:
Instead of `.format(table.label.value)` you can use f-strings (https://realpython.com/python-f-strings/):
```suggestion
                       f"Repeats on Seq table {table.label.value} should be set to '1', Malcolm has"
```

Reply:
You do know that you no longer need to review these, right :) ? But thanks, I think it looks tidier with f-strings.